app/uk/gov/hmrc/apisubscriptionfields/model/JsonFormatters.scala

```diff
@@ -36,16 +37,42 @@ trait SharedJsonFormatters {
 
 trait JsonFormatters extends SharedJsonFormatters {
 
+  implicit val ValidationRuleTypeReads = Reads.enumNameReads(ValidationRuleType)
+
+  val validationRuleReads: Reads[ValidationRule] = (
+    ((JsPath \ "validationRuleType").read[ValidationRuleType]) and
+      ((JsPath \ "value").read[String] or Reads.pure(""))
```

Comment:
This should surely be option in the case class if we really expect it to allow blanks.  Otherwise the user should be forced to enter some text.